### ISSUE TITLE
CMS screen profileID "garbage" crashes Geeqie

### ISSUE THREAD

**Issue by mowgli** [BestImageViewer/geeqie#363]
With LCMS1, Geeqie 1.1 on terminal prints warnings such as

(geeqie:14768): Gtk-WARNING **: Failed to set text from markup due to error 
parsing markup: Error on line 2 char 49: Invalid UTF-8 encoded text in name - not valid 'Image profile: sRGB
Screen profile:  - \xf8\x99\x90\x81\x82\xf8\x94\x90\x81\xae\xfd\xac\x8b\xb0\x80\x80\xfd\xac\x98\x96\x8d\xaf\xfd\xae\x99\x92\xbd\xa5'

With LCMS2, it additionally crashes in g_markup().

The culprit is the profileID MD5 value that's being printed as a tooltip in layout_utils.c.

Reported by: mschwendt

**Comment by mowgli** [comment]
Original patch:
```patch
diff -Nurb --strip-trailing-cr geeqie-1.1-orig/src/layout_util.c geeqie-1.1/src/layout_util.c
--- geeqie-1.1-orig/src/layout_util.c	2012-08-12 22:13:41.000000000 +0200
+++ geeqie-1.1/src/layout_util.c	2014-05-28 12:55:30.080458921 +0200
@@ -2313,7 +2313,7 @@
 	if (layout_image_color_profile_get_status(lw, &image_profile, &screen_profile))
 		{
 		gchar *buf;
-		buf = g_strdup_printf(_("Image profile: %s\nScreen profile: %s"), image_profile, screen_profile);
+		buf = g_strdup_printf(_("Image profile: %s"), image_profile);
 		g_object_set(G_OBJECT(action), "tooltip", buf, NULL);
 		g_free(image_profile);
 		g_free(screen_profile);
```

**Comment by caclark** [comment]
Fixed in commit a1afabd3ce
